### ISSUE TITLE
Add config option to limit embystat sidepanel

### ISSUE THREAD

**Issue by ghost** [mregni/EmbyStat#1027]
There are many options on the sidebar that won't be useful to most users.

It would be great if there was a preview mode, which would have embystat only show the movies and shows sections and nothing else. This would allow people to preview what you have in your library.

Embystat looks great so far thank you

**Comment by mregni** [comment]
Thanks for your feedback. Can you explain to me a bit more in detail what you mean with "a preview mode"? Like a tooltip when you hover over the menu item? Or really expanding the menu item with some very basic data (movie count, size, ...)?

**Comment by ghost** [comment]
Sorry for the unclear explanation. What I'm referring to is a mode that would allow others to see what is in your library, i.e. Movies and TV shows. You would not want reveal anything about your users, your server instance, etc. 

The use case is this: You want to show your friends your library, before creating an account for them.

**Comment by mregni** [comment]
Haaa, some kind of "guest" view where no sensitive data is shared? Sounds like a good idea. But for that, I need to implement the user login stuff first :)

**Comment by ghost** [comment]
Yep, but user login isn't necessary for this feature. You could just have a boolean in the config, if guest=true, then hide (list of things that are sensitive). The admin would have to manually change this config item.

And then later on once the user login feature has been implemented, you would set guest=false once the user has logged in.

**Comment by mregni** [comment]
I will have to add some more config as well di disable or enable this feature completely. I'll draw some ideas for this feature and come back to you :)
